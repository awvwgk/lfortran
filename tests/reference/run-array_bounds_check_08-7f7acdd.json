{
    "basename": "run-array_bounds_check_08-7f7acdd",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_08.f90",
    "infile_hash": "5b4ee4c8e59df328ecfe694741c18176b47b4000ad17c194947834c4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_08-7f7acdd.stderr",
    "stderr_hash": "4a491feca4eeb1f2e98534048c43a8d2b19454ae0b22ccb4226d7c2f",
    "returncode": 1
}