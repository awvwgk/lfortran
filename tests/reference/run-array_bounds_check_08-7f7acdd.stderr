At 4:3 of file tests/errors/array_bounds_check_08.f90
Runtime error: Array shape mismatch in subroutine 'ff'

Tried to match size 5 of dimension 1 of argument number 1, but expected size is 100000
